@xen-orchestra/lite/src/locales/fr.json

```
@@ -108,25 +109,31 @@
   "host.active": "Actif | Actif | Actifs",
   "host.inactive": "Inactif | Inactif | Inactifs",
 
+  "host-internal-network": "Hébergement de réseaux internes",
+
   "invalid-field": "Champ invalide",
   "keep-me-logged": "Rester connecté",
   "keep-page-open": "Ne pas rafraichir ou quitter cette page avant la fin du déploiement.",
   "language": "Langue",
   "last-week": "Semaine dernière",
   "load-now": "Charger maintenant",
   "loading-hosts": "Chargement des hôtes…",
+  "locking-mode": "Mode verouillé",
+  "locking-mode-default": "Mode verouillé par defaut",
```

Same here, maybe the wording can be improved?
BTW, some typos need to be fixed:
```suggestion
  "locking-mode": "Mode verrouillé",
  "locking-mode-default": "Mode verrouillé par défaut",
```

---

Any suggestion ? :smiley: